content/stub.compose-ui.js

```diff
@@ -684,8 +684,9 @@ ComposeSession.prototype = {
       draft: function ({ to, cc, bcc }) {
         let makeTokens = function (aList) {
           let [list, listEmailAddresses] = parse(aList);
-          return [asToken(null, item, listEmailAddresses[i], null)
-            for each ([i, item] in Iterator(list))];
+	  return Array.prototype.map.call(list, function(item, i) {
```

Comment:
My bad, for-of doesn't give you the index when iterating on an array. So yeah there are some cases where you do have to use map.